modules/settings-provider.js

```diff
@@ -9,7 +9,7 @@ const DEFAULT_CONVENTIONAL_COMMITS = {
         "release": "minor"
     },
```

Comment:
the release values accepted are "major,minor and patch" those are used to calculate the next release version:

![image](https://user-images.githubusercontent.com/97619732/183880503-1527bda5-102a-4780-9d35-a7c36dc2a971.png)